lib/cdo/solr_helper.rb

```diff
@@ -75,29 +80,21 @@ def self.update_document_type_user(solr_server, db_id, solr_id)
       solr_user_doc['create_ip_location_p'] = "#{location.latitude},#{location.longitude}" if location.latitude && location.longitude
     end
 
-    begin
-      solr_server.update([solr_user_doc])
-    rescue Exception => e
-      puts "EXCEPTION OCCURRED: #{e.message}"
-    end
+    solr_server.update([solr_user_doc])
```

Comment:
Now that this is going to be run via cron rather than locally, I think it makes sense to simply ignore the exception and let it feed into Honeybadger. Note that, previously, the exception was caught and *not* reraised.